content/docs/querying/operators.md

```diff
@@ -50,19 +56,18 @@ either `0` (`false`) or `1` (`true`), depending on the comparison result.
 value of every data sample in the vector, and vector elements between which the
 comparison result is `false` get dropped from the result vector.
 
-**Between two instant vectors**, these operators behave as a filter: They apply
-to vector elements that have identical sets of labels between the two vectors.
-Vector elements for which the expression evaluates to `false` or which don't
-find an exact label match on the other side of the expression get dropped from
-the result, while the others get carried over into a result vector with their
-original (left-hand-side) metric names and data values.
+**Between two instant vectors**, these operators behave as a filter, applied to
+matching entries. Vector elements for which the expression evaluates to false or
```

Comment:
"isn't true" rather than "evaluates to false", as otherwise we'd need to explain how the operators work internally